app/controllers/register.js

```diff
@@ -0,0 +1,20 @@
+const Register = require('../models/register');
+const errors = require('../errors');
+
+exports.list = (req, res) =>
+  Register.find().then((register) => res.json(register));
+
+exports.delete = async (req, res) => {
+  const register = await Register.findOne({
```

Comment:
No, if you try to delete a register that does not exist this will toss a 500 error. It's kinda unrealistic to
1) fetch a list of registrations
2) delete one of them
3) and that registration doesn't exist

But I can make it toss a 404 and not 500